ui/component/publishFormErrors/view.jsx

```diff
@@ -28,7 +28,7 @@ function PublishFormErrors(props: Props) {
       {uploadThumbnailStatus === THUMBNAIL_STATUSES.IN_PROGRESS && (
         <div>{__('Please wait for thumbnail to finish uploading')}</div>
       )}
-      {!!editingURI && !isStillEditing && !filePath && (
```

Comment:
?

Reply:
Fixed this, I noticed there are some typos or errors using `editingURI`, I'll keep debugging